xml/System/Environment.xml

```diff
@@ -1753,8 +1753,8 @@ To identify the operating system platform, for example, Linux or Windows, you ca
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the number of processors on the current machine.</summary>
-        <value>The 32-bit signed integer that specifies the number of processors on the current machine. There is no default. If the current machine contains multiple processor groups, this property returns the number of logical processors that are available for use by the common language runtime (CLR).</value>
+        <summary>Gets the number of processors on the current machine or container (like a Docker container).</summary>
+        <value>The 32-bit signed integer specifies the number of processors that are available. There is no default. If the current machine contains multiple processor groups, this property returns the number of logical processors that are available for use by the common language runtime (CLR).</value>
```

Comment:
```suggestion
        <value>The 32-bit signed integer specifies the number of processors that are available.</value>
```
Move the comment about processor groups into Windows specific sections in remarks, or just delete it - it is not very relevant.

Reply:
The value's description should read either "integer **that** specifies..." or "integer specif**ying**...".  I agree on the rest. "There is no default" makes no sense to me.

Reply:
The wording reads well to me.